bin/utest/pipe.c

```diff
@@ -15,15 +16,19 @@
 
 static sig_atomic_t signal_delivered;
 
-static void sigpipe_handler(int signo) {
-  assert(signo == SIGPIPE);
-  signal_delivered = 1;
+static void handler(int signo) {
+  if (signo == SIGPIPE) {
+    assert(signo == SIGPIPE);
+    signal_delivered = 1;
+  }
+  if (signo == SIGALRM) {
+  }
```

Comment:
What with `SIGALRM`?

Reply:
What do you mean?
I use it to simply wake up from interruptible sleep.

Reply:
What is a purpose of that if statement?

Reply:
OK, I get what you mean